Dockerfile

```diff
@@ -22,78 +22,80 @@
 # 7. copy ALL of REMOTE_SOURCES to REMOTE_SOURCES_DIR, not just upstream1/cachito.env and upstream1/app/registry-ca.pem
 # 8. add Brew metadata
 
-# Stage 1 - Install dependencies
+# Stage 1 - Build nodejs skeleton
 #@follow_tag(registry.redhat.io/ubi9/nodejs-18:1)
-FROM registry.access.redhat.com/ubi9/nodejs-18:1 AS deps
+FROM registry.access.redhat.com/ubi9/nodejs-18:1 AS skeleton
 USER 0
 
+# Install isolate-vm dependencies
+RUN dnf update -y && \
+  dnf install -y zlib-devel brotli-devel && \
+  dnf clean all
```

Comment:
if this is a temporary stage, you could skip cleanup

Reply:
Good point